src/transformers/tokenization_utils_base.py

```diff
@@ -1621,34 +1621,20 @@ def from_pretrained(cls, pretrained_model_name_or_path, *init_inputs, **kwargs):
                         resume_download=resume_download,
                         local_files_only=local_files_only,
                     )
-        except EnvironmentError:
-            if pretrained_model_name_or_path in s3_models:
-                msg = "Couldn't reach server at '{}' to download vocabulary files."
-            else:
-                msg = (
-                    "Model name '{}' was not found in tokenizers model name list ({}). "
-                    "We assumed '{}' was a path or url to a directory containing vocabulary files "
-                    "named {}, but couldn't find such vocabulary files at this path or url.".format(
-                        pretrained_model_name_or_path,
-                        ", ".join(s3_models),
-                        pretrained_model_name_or_path,
-                        list(cls.vocab_files_names.values()),
-                    )
-                )
-
-            raise EnvironmentError(msg)
+                except requests.exceptions.HTTPError as err:
+                    if "404 Client Error" in str(err):
+                        logger.debug(err)
+                        resolved_vocab_files[file_id] = None
```

Comment:
We don't need this check in the modeling utils?

Reply:
No, because tokenizers' are the only files where we have this behavior where we try to load multiple files, some of which can be missing. For models/config you need the one actual file.